lib/experimental/new_embed.dart

```diff
@@ -533,7 +624,10 @@ class NewEmbedTabController extends TabController {
   }
 
   void setTabVisibility(String tabName, bool visible) {
-    TabElement tab = tabs.firstWhere((t) => t.name == tabName);
+    var tab = tabs.firstWhere((t) => t.name == tabName, orElse: () => null);
+    if (tab == null) {
```

Comment:
Using conditional prop access on the call to toggleAttr would save a few lines here.

Reply:
good idea - done